File: 백준/Bronze/2525. 오븐 시계/오븐 시계.py
h, m = map(int,input().split())
time = int(input())

if m + time >= 60:
    h = h + (m + time) // 60
    m = (m + time) % 60
    if h >= 24:
        h = h-24
else:
    m = m+time
    
print(h,m)

File: 백준/Bronze/2525. 오븐 시계/오븐 시계.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int h = sc.nextInt();
        int m = sc.nextInt();
        int time = sc.nextInt();
        h = (h +  (m + time) / 60) % 24;
        m = (m + time) % 60;
        System.out.println(h + " " + m);
    }
}
